### DISCUSSION TITLE
Is it possible to migrate a V4.1 project to V5?

### DISCUSSION THREAD

**Discussion by alannaidon** [adonisjs/core#3064] [Help] [answered]
Hi! Today I have 2 APIs and both are running Adonis V4.1  

I really want to use `Replica` that comes as 'first-class' with **Adonis V5**.  
Is it possible to migrate an existing project? I tried to follow [this guide](https://docs.adonisjs.com/releases/out-of-preview), but too many errors and seemed impossible to do.  

Also, the security updates will be given only 'till the end of 2021. What you guys recommend?

**Comment by McSneaky** [comment, marked as answer]
That guide is meant to be followed when upgrading from V5-preview to V5 stable

I'd reccommend to just create new V5 project with `create-adonis-ts-app` and then move things over manually

V4 to V5 is quite big change. You can still copy-paste most of your code logic etc tho. Just have to adjust it to V5 file by file

**Reply by alannaidon** [reply]
> That guide is meant to be followed when upgrading from V5-preview to V5 stable
> 
> I'd reccommend to just create new V5 project with `create-adonis-ts-app` and then move things over manually
> 
> V4 to V5 is quite big change. You can still copy-paste most of your code logic etc tho. Just have to adjust it to V5 file by file

Thank you, sir!
